### ISSUE TITLE
update usage commands in docs

### ISSUE THREAD

**Issue by marc0olo** [aeternity/aepp-cli-js#216]
they seem to be outdated right now.

following is printed on my end:

```
marc0olo@DESKTOP-B1QLEDL:~/git/aeternity/AEXs$ aecli help
Usage: aecli [options] [command]

Options:
  -V, --version   output the version number
  -h, --help      display help for command

Commands:
  config          Print the sdk default configuration
  chain           Interact with the blockchain
  inspect         Get information on transactions, blocks,...
  account         Handle wallet operations
  contract        Compile contracts
  name            AENS system
  tx              Transaction builder
  oracle          Interact with oracles
  crypto          Crypto helpers
  help [command]  display help for command
  ```

**Comment by marc0olo** [comment]
I think this could be sth. for hacktoberfest @davidyuk :D

**Comment by davidyuk** [comment]
Why do you think this is outdated? Basically, it is a list of cli modules with some description, you can request additional help for each of them, for example
```
./bin/aecli.mjs help crypto
Usage: aecli crypto [options] [command]

Options:
  -h, --help                     display help for command

Commands:
  decode <base58address>         Decodes base58 address to hex
  sign [options] <tx> [privkey]
  unpack <tx>
  help [command]                 display help for command
```

**Comment by marc0olo** [comment]
I was referring to the README file where it is outdated when I remember. can't 100% recall right now

**Comment by davidyuk** [comment]
got it, should be fixed in #217
